lib/commands/render.js

```diff
@@ -90,7 +91,10 @@ module.exports = function(parser, tilelive) {
 
       require("../modules")(tilelive, opts);
 
-      return tilelive.load(opts.source, function(err, source) {
+      var source   = url.parse(opts.source);
+      source.query = {scale: opts.scale}
```

Comment:
Now that the query params are pulled above, don't override any that are present (and respect a previously-set `scale` on the source):

```javascript
source.query.scale = source.query.scale || opts.scale;
```